core-data/test/resolvers.js

```diff
@@ -46,3 +46,24 @@ describe( 'getEntityRecord', () => {
 		expect( received ).toEqual( receiveEntityRecords( 'root', 'postType', POST_TYPE ) );
 	} );
 } );
+
+describe( 'getEntityRecords', () => {
+	const POST_TYPES = {
+		post: { slug: 'post' },
+		page: { slug: 'page' },
+	};
+
+	beforeAll( () => {
+		apiRequest.mockImplementation( ( options ) => {
+			if ( options.path === '/wp/v2/types?context=edit' ) {
+				return Promise.resolve( POST_TYPES );
+			}
+		} );
+	} );
+
+	it( 'yields with requested post type', async () => {
+		const fulfillment = getEntityRecords( {}, 'root', 'postType' );
+		const received = ( await fulfillment.next() ).value;
+		expect( received ).toEqual( receiveEntityRecords( 'root', 'postType', Object.values( POST_TYPES ) ) );
+	} );
+} );
```

Comment:
Would it be worthwhile to have taxonomy tests here too?

Reply:
I don't think so. I think it's worthwhile is to add tests to ensure the selectors/resolvers are generated (the code in the `core-data/index.js`) but several parts of this code are already tests: the methodName and the underlying selectors/resolvers.

Reply:
👍